README.md

```diff
@@ -1 +1,128 @@
-# JS-best-practices-todo-list-code-review
\ No newline at end of file
+# JS-best-practices-todo-list-code-review
+
+<a name="readme-top"></a>
+
+# 📗Table of Contents
+
+- [JS-best-practices-todo-list-code-review](#js-best-practices-todo-list-code-review)
+- [📗Table of Contents](#table-of-contents)
+- [📖 \[Interactive-to-do-list\] ](#-interactive-to-do-list-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+  - [👥 Author ](#-author-)
+    - [Future Features ](#future-features-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Interactive-to-do-list] <a name="about-project"></a>
+
+**[Interactive-to-do-list]** is a project that keeps a list of the tasks that the user needs to complete in a day. You can add tasks, delete tasks, mark tasks as complete, save tasks for future reference
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML</li>
+    <li>CSS</li>
+    <li>JavaScript</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- An HTML File
+- A CSS File
+- A Javascript file
+- An md file
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>
+
+- [Live Demo Link]()
```

Comment:
> Please add a live demo link to your project. It will help give users a preview of the project.

Reply:
Thank you for taking note of this omission, I will definitely include the live link in the README file now